js/main.js

```diff
@@ -1,14 +1,16 @@
 const $photoInput = document.querySelector('.photo-input');
+const $img = document.getElementById('img');
 function updateValue() {
-  document.getElementById('img').setAttribute('src', $photoInput.value);
+  $img.setAttribute('src', $photoInput.value);
 }
 $photoInput.addEventListener('input', updateValue);
 
 const $submit = document.querySelector('#form');
+const $imgPlaceholder = document.getElementById('img');
```

Comment:
You've already queried for this on line 2, so you shouldn't do it again. Just utilize your `$img` variable